rust/catalyst-signed-doc-macro/src/rules/content_type.rs

```diff
@@ -24,18 +27,24 @@ pub(crate) fn into_rule(
     }
 
     if matches!(field.required, IsRequired::Yes) {
-        anyhow::ensure!(!field.value.is_empty(), "'value' field must exist");
+        anyhow::ensure!(!is_field_empty, "'value' field must exist");
     }
 
-    let template = ContentTypeTemplate(field.value.clone());
-    let Some(_) = content_types.get(&template) else {
-        return Err(anyhow::anyhow!("Unsupported Content Type: {}", field.value));
-    };
-    let ident = template.ident();
-
-    Ok(quote! {
-        crate::validator::rules::ContentTypeRule::Specified {
-            exp: ContentType::#ident,
-        }
-    })
+    if let Some(value) = &field.value {
```

Comment:
if `value` field is missing but `field.required != excluded` it should be an error 

```rust
let Some(value) = &field.value else {
    anyhow::bail!("'value' field must exist");
};
```